src/components/Layout/Navbar/NavbarDesktop.tsx

```diff
@@ -45,23 +37,17 @@ export default function NavbarDesktop() {
    <nav className={styles.navbar}>
      <UnorderedList
       listStyleType="none"
-      sx={{
-        color: "white",
-        background: "#1d1283",
-        display: "flex",
-        padding: "20px",
-        margin: 0,
-        gap: "20px",
-        alignItems: "center",
-      }}
+      className={styles.navbarMenu}
     >
       <ListItem>
+        <Link href={HOME_URL}>
```

Comment:
Not sure if Link component has any benefit here, the link isn't pointing to another route, rather different url.